Model/Page.php

```diff
@@ -174,6 +174,11 @@
     protected static $slugifyMethod;
 
     /**
+     * @var int
+     */
+    private $targetRedirectType = 302;
```

Comment:
Maybe add this constant? https://github.com/symfony/symfony/blob/3bbb657560615b80c8a9b3b6da8602349bd3fbd9/src/Symfony/Component/HttpFoundation/Response.php#L36